docs/guide/hello-world.md

```diff
@@ -101,7 +101,8 @@ You can read more about ES6 modules from Axel Rauschmayer's "[_ECMAScript 6 modu
 
 Now it's time to configure our section. Open `client/sections.js` and add the following code to the end of the `sections` array:
 
-```javascript
+```
```

Comment:
>ESLint says there's some sort of unexpected token here, but there isn't.

I think what's going on here is the JavaScript interpreter is trying to interpret this block as valid JavaScript, but it's not (because it's just an object -- there's no command, etc.) The "unexpected token" is the starting `{`.

All that to say... I think removing the syntax highlighting is a great call! Because it's not actually JavaScript! 👍